recipes/occt/yum_requirements.txt

```diff
@@ -0,0 +1,4 @@
+libXt-devel
+libXmu-devel
+libXi-devel
+mesa-libGLU-devel
```

Comment:
linux is skipped because it already passed the build once. with toolchain travis looks also good. So next is windows. I will change the skips once the compilation on windows works.